### PR TITLE
Incorporate docs feedback

### PR DESCRIPTION
**Pull request by CyberHoward**
Incorporate some feedback on our documentation. This includes: 
1. Adding a section for devs coming from EVM
2. Updating much of our graphs to differentiate between User and Owner calls
3. Update the dependency assertion graph
4. Elaborate on why the Proxy and Manager are different contracts
5. Re-instate the section on the Oracle functionality.

### CODE REVIEW

File: framework/docs/src/5_platform/7_oracle.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Account Value Oracle
+
+Have you ever wished to know the value of your portfolio? I'm guessing you have. Luckily Abstract has an integrated system to retrieve the value of your assets, on-chain!
```
Comment (adairrr) [inline framework/docs/src/5_platform/7_oracle.md:3]: 🔥🔥

File: framework/docs/src/5_platform/7_oracle.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Value is relative
+
+The value of something is always relative to some other thing. A bitcoin isn't valued at 20,000, it's valued at 20,000 **USD**. Likewise the first decision that's required to work with Abstract's oracle is: In what currency do you want your Account's assets to be valued?
+
+We'll call this currency the *base asset*. There can never be more than one base asset and every asset will be valued in terms of this base asset.
```
Comment (adairrr) [inline framework/docs/src/5_platform/7_oracle.md:9]: We should probably mention that this feature is almost always for application developers and not necessarily user accounts?

File: framework/docs/src/3_framework/0_prerequisites.md [hunk 3]
```diff
@@ -75,7 +75,19 @@ The application of the actor model in the CosmWasm framework provides the follow
   prevents
   inconsistent states in the contract.
 
-To learn more about CosmWasm, check out the <a href="https://book.cosmwasm.com/" target="_blank">official
+#### Coming from EVM?
+
+There are a few key differences between the EVM and CosmWasm that you should be aware of. The most important one is that instances of contracts and the code that they run against are two different concepts in CosmWasm. This means that you can have multiple instances of the same contract code running at the same time, each with their own state. This is not possible in EVM, where the contract code and the contract instance are the same thing.
```
Comment (adairrr) [inline framework/docs/src/3_framework/0_prerequisites.md:80]: We can mention that this leads to significantly reduced gas cost in instantiating new contracts and in complex contract interactions.

Comment (CyberHoward) [reply framework/docs/src/3_framework/0_prerequisites.md:80]: I'd rather keep this section factual but will add a link to comparison article if they want to deep-dive the differences and advantages!

**Pull request merged**